### PR TITLE
DAG-1746: Forward dependencies from task definitions

### PR DESCRIPTION
**Pull request by dbradf**

### CODE REVIEW

File: docs/generating_tasks.md [hunk 1]
```diff
@@ -157,3 +157,82 @@ sub-tasks generated from a task definition into a single display task.
...
+
+A sample file would look like this:
+
+```yaml
+project: mongodb-mongo-master
+revision: abc123
+version_id: 321abc
+```
+
+You much provide the expansion file when running the `mongo-task-generator` command:
```
Comment (MikhailShchatko) [inline docs/generating_tasks.md:188]: `must` instead of `much`?

**Review by MikhailShchatko — APPROVED** [review]
LGTM with a comment

File: docs/generating_tasks.md (hunk 1 above)
Comment (dbradf) [reply docs/generating_tasks.md]: fixed.

**Comment by dbradf** [comment]
evergreen merge

**Pull request merged**